src/org/labkey/test/components/SetDefaultPageClass.java

```diff
@@ -0,0 +1,19 @@
+package org.labkey.test.components;
+
+import org.labkey.test.Locator;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+
+public class SetDefaultPageClass extends LabKeyPage
```

Comment:
Page class names should correspond to the server action they wrap (ViewPreferencesAction) and the package name should correspond with the module containing that action (study).
```suggestion
package org.labkey.test.pages.study;

import org.labkey.test.Locator;
import org.labkey.test.pages.LabKeyPage;
import org.openqa.selenium.WebDriver;

public class ViewPreferencesPage extends LabKeyPage
```

Reply:
ok